Loop/Managers/DeviceDataManager.swift

```diff
@@ -1337,7 +1337,7 @@ extension DeviceDataManager {
         let command: RemoteCommand
         
         do {
```

Comment:
Pass the defaultAbsorptionTime when no absorption given in remote command, rather than just using 3.0.